### PR TITLE
Commands links to documentations 

### PR DESCRIPTION
**Pull request by AvitalFineRedis**

### CODE REVIEW

**Comment by chayim** [comment]
Have you run _invoke build-docs_ or looked at the documents on the web? To do so run _invoke docserver_. We generate a lot of warnings because of the "More information here" text.  Each _here_ needs to be a separate string, in order to silence the warnings. How about instead, we use something like "module <command name> documentation", replacing each command_name with the appropriate item (i.e cfdump, cfdel, etc).

```
/data/repos/redis/redisplus-py/redisplus/ai/commands.py:docstring of redisplus.ai.commands.CommandMixin.dag:30: WARNING: Inline strong start-string without end-string.
/data/repos/redis/redisplus-py/redisplus/ai/commands.py:docstring of redisplus.ai.commands.CommandMixin.dag:30: WARNING: Inline strong start-string without end-string.
/data/repos/redis/redisplus-py/redisplus/ai/commands.py:docstring of redisplus.ai.commands.CommandMixin.dag:30: WARNING: Inline strong start-string without end-string.
/data/repos/redis/redisplus-py/redisplus/ai/commands.py:docstring of redisplus.ai.commands.CommandMixin.tensorset:26: WARNING: Duplicate explicit target name: "here".
/data/repos/redis/redisplus-py/redisplus/ai/commands.py:docstring of redisplus.ai.commands.CommandMixin.tensorset:26: WARNING: Duplicate explicit target name: "here".
/data/repos/redis/redisplus-py/redisplus/ai/commands.py:docstring of redisplus.ai.commands.CommandMixin.loadbackend:19: WARNING: Unexpected section title or transition.
<snip>
```

**Comment by AvitalFineRedis** [comment]
@chayim 
Yes. I did try _invoke docserver_ and it worked for me

**Comment by chayim** [comment]
Awesome. I caught the warnings in the terminal when testing that way.

File: redisplus/graph/commands.py [hunk 1]
```diff
@@ -129,12 +145,14 @@ def profile(self, query):
...
         """
         return self.query(query, profile=True)
 
     def slowlog(self):
         """
         Get a list containing up to 10 of the slowest queries issued against the given graph ID.
+        For more information see `GRAPH.SLOWLOG <https://oss.redis.com/redisgraph/master/commands/#graphslowlog>`_.
 
         Each item in the list has the following structure:
         1. A unix timestamp at which the log entry was processed.
```
Comment (AvitalFineRedis) [inline redisplus/graph/commands.py:158]: The numbers do not look right on the docserver :\

**Pull request merged**